frontend/kubecloud/src/views/RegisterVerifyView.vue

```diff
@@ -49,17 +49,31 @@
 </template>
 
 <script setup lang="ts">
-import { ref, reactive } from 'vue'
+import { ref, reactive, onMounted } from 'vue'
 import { useRoute, useRouter } from 'vue-router'
 import { authService } from '../utils/authService'
-import { userService } from '@/utils/userService'
+import { useNotificationStore } from '@/stores/notifications'
 import { useUserStore } from '@/stores/user'
 
 const route = useRoute()
 const router = useRouter()
 
+// Check if we have valid registration data and pre-fill email
+onMounted(() => {
+  // Try to get email from stored registration data
+  if (authService.hasTempRegistrationData()) {
+    const storedData = (authService as any).getTempRegistrationData()
```

Comment:
Add getTempRegistrationData public method in authService and use it in here to avoid using any.